.github/workflows/embedded-sdk-release.yml

```diff
@@ -0,0 +1,40 @@
+name: embedded-sdk-release-workflow
+
+on:
```

Comment:
Hopefully this workflow doesn't have any issues, or I'll need a follow-up PR to fix it. Reviewers please help me check this! 🙏

Reply:
I tested and got it working on my fork, so I can be pretty confident that it will work here.